Assets/Live2D/Cubism/Framework/ObjectExtensionMethods.cs

```diff
@@ -14,7 +14,7 @@ namespace Live2D.Cubism.Framework
     /// <summary>
     /// Extensions for <see cref="Object"/>s.
     /// </summary>
-    internal static class ObjectExtensionMethods
+    public static partial class ObjectExtensionMethods
```

Comment:
```suggestion
    public static class ObjectExtensionMethods
```

We have suggested one modification to align with our coding conventions. Once that change is complete, we will be able to incorporate it.